packages/renderer/src/lib/ingresses-routes/IngressRouteDetailsSummary.svelte

```diff
@@ -2,14 +2,18 @@
 import type { V1Ingress } from '@kubernetes/client-node';
 import type { V1Route } from '../../../../main/src/plugin/api/openshift-types';
 import ErrorMessage from '../ui/ErrorMessage.svelte';
-import type { IngressUI } from './IngressUI';
-import type { RouteUI } from './RouteUI';
-import IngressRouteColumnHostPath from './IngressRouteColumnHostPath.svelte';
-import IngressRouteColumnBackend from './IngressRouteColumnBackend.svelte';
+import ObjectMeta from '../kube/details/ObjectMeta.svelte';
+import Ingress from '../kube/details/Ingress.svelte';
+import IngressStatus from '../kube/details/IngressStatus.svelte';
```

Comment:
need to find a new home for this objects

it looks strange to import from `../kube/details` when we're inside `ingresses-routes`

Reply:
also we should favor the `/@/ notation`